### ISSUE TITLE
Urgently fixed in this release: Inconsistency scanning result

### ISSUE THREAD

**Issue by ellyasafaa** [onaio/rdt-standard#796]
There is inconsistency in the heat sensor result when scanning the barcode. Please kindly take a look at the issue in the video:
https://drive.google.com/file/d/1-BXtki2ZqfTvF75W3RckLuBFu0hIKWm6/view?usp=sharing

@syedowaisali90 @mwkhan1983 @vincent-karuri

**Comment by vincent-karuri** [comment]
The scanner reads an RDT as invalid if the sensor is triggered or the RDT is expired. I'm not sure why but that one moment, it probably detected the sensor as triggered.

Also, is the barcode correctly encoded? You should be able to see a preview of the RDT that should be captured if the RDT is available in the capture library of the app.

**Comment by ellyasafaa** [comment]
@vincent-karuri Yes, the RDT has been correctly encoded. I have checked all information and checked the condition of the barcode that is not activated by the heat. There will be other signs as well if the RDT is triggered by the heat, like black dots in the barcode, so the barcode is fine. For the capturing image, I am aware of that, and it is not available yet in the library, so it won't be captured, but for the sensor, I think it should display consistent result, right?

**Comment by ellyasafaa** [comment]
Uploading WhatsApp Video 2021-03-03 at 5.00.54 PM.mp4…

RDT Antibody

**Comment by ellyasafaa** [comment]
https://user-images.githubusercontent.com/73918009/109795923-1e238300-7c4a-11eb-81f8-1b2abcbbdcb4.mp4

on Antibody RDT - AllTest

**Comment by vincent-karuri** [comment]
@ellyasafaa OneScan developers and myself have been unable to reproduce this both when using the OneScan app directly and when using the integrated version on the CTS app.

After discussing with Zebra and as a follow up could you provide: 

- version of CTS app this is tested on
- version of OneScan app installed on the device
- exact barcode sample that caused the problem 
- clean and direct explanation of reproducible conditions under which the issue appears.

We will share a test apk from Zebra that calls directly into the OneScan app. Try to reproduce and document the issue with both the Zebra apk and the CTS app.

**Comment by vincent-karuri** [comment]
![41c2929b-8653-41ad-91b3-9b6cb267e4c1](https://user-images.githubusercontent.com/33488286/110771925-1d06dd00-826c-11eb-9279-45ff415f53a0.jpg)

This is the barcode we tested on (which was shared as problematic by SID) but which was always marked as valid in multiple trials on CTS and OneScan apks.

**Comment by mwkhan1983** [comment]
@ellyasafaa apk to test this issue
https://drive.google.com/file/d/1zQNHhE9kObzunW46jz5J3XZez-5G3n85/view?usp=sharing

**Comment by ellyasafaa** [comment]
- version of CTS app this is tested on v1.9.1-playstore-release-CONFIGURABLE-RDT-SID (built on 12 Mar 2021)
- version of OneScan app installed on the device is onescan-0.9.54-demo-basic
- exact barcode sample that caused the problem: The picture on the drive folder shared below. There are 6 barcodes that have the same issue and replicated both on ONA Mock App and CTS App. On ONA Mock App, it is written as status: Ok, but the sensor triggered said yes. On the folders, there are numbers I wrote down in the video file name when I used the ONA Mock App to scan the barcode showing the number of scanning when the sensor triggered said 'yes'. For example, 4,5 Ab RDT ONA Mock App means that the 4th and 5th scan using ONA Mock App showed that the sensor triggered said 'yes', although the status: Ok. Please kindly take a look at that. For the antibody with the barcode of 420YV-6, we cannot try it on ONA Mock App since the camera button on the device cannot be clicked.
- clean and direct explanation of reproducible conditions under which the issue appears
I have looked at how it can be reproduced, but I don't find a specific condition when it can be reproduced. It just randomly happened. I thought at first the position of the user when they scan the barcode or the specific scanning location in the barcode will affect the scanning result. However, I don't think that is the problem. You can take a look at the video to get a better understanding of how the issue can appear. In the user side, we are just doing scanning without any specific step on the apps, just click the RDT form and follow the steps to get into the scanning page, and after scanning, the result is not consistent.

The detail for each video and the barcode used:
https://drive.google.com/drive/folders/1aMxdE4WTwRsuUVqgUFhDEsvf1JqbHyms?usp=sharing

On the other hand, I found crashes multiple times. Will open a new issue for that.

**Comment by vincent-karuri** [comment]
thanks @ellyasafaa sharing this with Zebra
